example_data/sample_bundle.json

```diff
@@ -528,21 +634,37 @@
       }
     },
     {
-      "fullUrl": "PlanDefinition/38018b47-b29a-8b06-daf6-6c5f2577bffa",
+      "fullUrl": "PlanDefinition/20RG-205G0047-plan-definition",
       "resource": {
         "resourceType": "PlanDefinition",
-        "id": "38018b47-b29a-8b06-daf6-6c5f2577bffa",
+        "id": "20RG-205G0047-plan-definition",
         "status": "active",
+        "identifier": [
+          {
+            "value": "20RG-205G0047-plan-definition"
+          }
+        ],
+        "description": "Sequence variant screening in Maria321 BrighGosh321 because of seizure disorder and global developmental delay.",
```

Comment:
we didn't have this here before, is this intentional to move it from where we previously included it?

Reply:
We want to remove this from here and only have it in the ServiceRequest resource